azurerm/data_source_monitor_action_group_test.go

```diff
@@ -171,11 +229,119 @@ resource "azurerm_monitor_action_group" "test" {
+	}
+
+	azure_function_receiver {
+		name = "funcaction"
+		function_app_resource_id = "${azurerm_function_app.test.id}"
+		function_name = "myfunc"
+		http_trigger_url = "https://example.com/trigger"
+	}
+}
+
+resource "azurerm_automation_account" "test" {
+	name                = "acctestAA-%d"
+	location            = "${azurerm_resource_group.test.location}"
+	resource_group_name = "${azurerm_resource_group.test.name}"
+
+	sku {
+		name = "Basic"
+	}
+}
+
+resource "azurerm_automation_runbook" "test" {
+	name                = "Get-AzureVMTutorial"
+	location            = "${azurerm_resource_group.test.location}"
+	resource_group_name = "${azurerm_resource_group.test.name}"
+	account_name        = "${azurerm_automation_account.test.name}"
+	log_verbose         = "true"
+	log_progress        = "true"
+	description         = "This is an test runbook"
+	runbook_type        = "PowerShellWorkflow"
+
+	publish_content_link {
+		uri = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-automation-runbook-getvms/Runbooks/Get-AzureVMTutorial.ps1"
+	}
+}
+
+resource "azurerm_logic_app_workflow" "test" {
+	name                = "acctestLA-%d"
+	location            = "${azurerm_resource_group.test.location}"
+	resource_group_name = "${azurerm_resource_group.test.name}"
+}
+
+resource "azurerm_logic_app_trigger_http_request" "test" {
+	name         = "some-http-trigger"
+	logic_app_id = "${azurerm_logic_app_workflow.test.id}"
+
+schema = <<SCHEMA
+{
+	"type": "object",
+	"properties": {
+		"hello": {
+			"type": "string"
+		}
+	}
+}
+SCHEMA
+}
+
+resource "azurerm_storage_account" "test" {
+	name                     = "acctestSA-%d"
+	resource_group_name      = "${azurerm_resource_group.test.name}"
+	location                 = "${azurerm_resource_group.test.location}"
+	account_tier             = "Standard"
+	account_replication_type = "LRS"
+}
+
+resource "azurerm_app_service_plan" "test" {
+	name                = "acctestSP-%d"
+	location            = "${azurerm_resource_group.test.location}"
+	resource_group_name = "${azurerm_resource_group.test.name}"
+
+	sku {
+		tier = "Standard"
+		size = "S1"
+	}
+}
+
+resource "azurerm_function_app" "test" {
+	name                      = "acctestFA-%d"
+	location                  = "${azurerm_resource_group.test.location}"
+	resource_group_name       = "${azurerm_resource_group.test.name}"
+	app_service_plan_id       = "${azurerm_app_service_plan.test.id}"
+	storage_connection_string = "${azurerm_storage_account.test.primary_connection_string}"
 }
 
 data "azurerm_monitor_action_group" "test" {
   resource_group_name = "${azurerm_resource_group.test.name}"
   name                = "${azurerm_monitor_action_group.test.name}"
 }
-`, rInt, location, rInt)
+`, rInt, location, rInt, rInt, rInt, rInt/1e12, rInt, rInt)
```

Comment:
to be consistent with the other resources it'd make sense to pass in a short (e.g. 4/5 char) string here:

```suggestion
`, rInt, location, rInt, rInt, rInt, rString, rInt, rInt)
```